goal_src/jak2/pc/pckernel.gc

```diff
@@ -230,6 +230,11 @@
   (when (not (led-enabled? obj))
     (disable *led-fader-state*)
     )
+  
+   (if (-> *pc-settings* input-display?)
+        (input-display-on)
+        (input-display-off)
```

Comment:
I don't think this should be running once per frame, i'm not really concerned about performance problems but it's just not the idiomatic location for it.  It should only need to run when the user actually changes the setting as far as I can tell.

Reply:
tbh that makes so much sense

Reply:
So, after sleeping on it, the process still needs to be spun up on initialization no?